chain/jsonrpc/src/lib.rs

```diff
@@ -230,6 +233,21 @@ impl JsonRpcHandler {
             }
         }
 
+        #[cfg(feature = "ganache")]
+        {
+            let params = request.params.clone();
+
+            let res = match request.method.as_ref() {
+                // Ganache controls
+                "ganache_patch_state" => Some(self.ganache_patch_state(params).await),
+                _ => None,
+            };
+
+            if let Some(res) = res {
+                return res;
+            }
+        }
```

Comment:
There are two existing code patterns this might be modeled after -- the `adversarial` block above, and the normal block bellow. What's bellow looks like a better model here, for two reasons:

* the code just reads more "production ready". For example, parameter parsing happens in the caller, which improves readability and better layers error
* once we stabilize this, we would want to move it to the bellow block, as that's the pattern we use for "production" requests.

Reply:
done